tracer/src/Datadog.Trace.Tools.Runner/Aot/AotProcessor.cs

```diff
@@ -59,8 +59,9 @@ internal class AotProcessor
 
         static AotProcessor()
         {
-            Definitions = InstrumentationDefinitions.GetAllDefinitionsNative();
-            DerivedDefinitions = InstrumentationDefinitions.GetAllDerivedDefinitionsNative();
+            // TODO : Temporary build fix
+            Definitions = null; // InstrumentationDefinitions.GetAllDefinitionsNative();
+            DerivedDefinitions = null; // InstrumentationDefinitions.GetAllDerivedDefinitionsNative();
```

Comment:
I wonder if we should use the same json file for dependabot... wdyt?

Reply:
Yeah, quite possible, but I think the reality is that this isn't supported or used anywhere yet, so it's not worth worrying about until we come back to this IMO

Reply:
true